test/ApplicationTest.php

```diff
@@ -501,7 +507,7 @@ public function testOnDispatchErrorEventPassedToTriggersShouldBeTheOriginalOne()
     }
 
     /**
-     * @group 2981
+     * @group #2981
```

Comment:
Related to https://github.com/sebastianbergmann/phpunit/issues/4489

Reply:
Those should be removed. They predate repository split of 2.5 and are irrelevant